events/enigma/index.html

```diff
@@ -1,305 +1,263 @@
+                    </div>
+                </div>
+
+                <div class="event-poster-wrapper">
+                    <a href="#eventposter" role="button" data-toggle="modal">
+                        <div class="poster-overlay">
+                            <p>Click to enlarge.</p>
+                        </div>
+                        <figure class="events-thumbnail">
+                            <img src="poster.png" alt="Enigma Poster">
+                        </figure>
+                    </a>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-12">
+            <div class="upcoming-events">
+                <div class="upcoming-events-header">
+                    <h4>Problem Statement</h4>
+                </div>
+                <div class="upcoming-events-list ps">
+                    <div class="upcoming-event-wrap flex flex-wrap justify-content-between align-items-center">
+                        <header class="entry-header">
+                            <h4 class="entry-title">You will be given a dataset and 3-5 tasks on it (Exact number of problems will be released on site).
+                                 The solution of each problem will be evaluated and considered. The weights of each task shall be revealed at the competetion
+                                  site itslef. The participants are expected to develop an interface for easy input and output of the data. Top 8 teams shall
+                                   qualify for the second round involving a 4 minute presenttion and a short QnA. the event requires zero pre-event preparation
+                                    from the participant side in terms of data collection. Relevant theory material will be given to the participants.</h4>
+                        </header>
+                    </div>
+                    <div class="upcoming-events-header">
+                        <h4>Timeline</h4>
+                    </div>
+                    <div class="upcoming-event-wrap flex flex-wrap justify-content-between align-items-center">
+                        <header class="entry-header">
+                            <h4 class="entry-title">
+                                <ul class="ps-list">
+                                    <li>Participants must be present at the specified venue 10 minutes before the commencement of the event. Point distribution of tasks will be revealed.</li>                           
+                                    <li>15 minutes for the introduction of the problem statement and Q&A.</li>
+                                    <li>The duration of Enigma will be 5 hours.</li>
+                                    <li>Top teams will be given 10 minutes in the project presentation round.</li>
+                                    <li>Note: The times are subject to change depending on the number of participants.</li>
+                                </ul>
```

Comment:
Add something like, please ensure you have the development environments set up beforehand. 5 hours is less, you don't get time to do that!